setup.py

```diff
@@ -75,25 +75,26 @@ def get_version(version_file):
         # that you indicate whether you support Python 2, Python 3 or both.
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
         "Programming Language :: Python :: Implementation :: CPython",
     ],
-    python_requires=">=3.7",
+    python_requires=">=3.11",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords=["cyhy", "config"],
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data={"example": ["data/*.txt"]},
+    package_data={"cyhy_config": ["py.typed"]},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
-    install_requires=["docopt", "schema", "setuptools >= 24.2.0"],
+    install_requires=[
+        "boto3",
+        "pydantic",
+        "setuptools >= 24.2.0",
+    ],
     extras_require={
         "test": [
+            "boto3-stubs",
```

Comment:
Shouldn't the other stub dependencies listed in `.pre-commit-config.yaml` be listed here as well?

Reply:
This is something that may need to added to [cisagov/skeleton-python-library]() as well.  Should we do it in `test` or in a new `dev` section?

Reply:
See: bc903bbe142afbed2080abf756cce5501a06ac93

Reply:
Commit bc903bb is good enough for now, but I'd still like to get a consensus on [this question](https://github.com/cisagov/cyhy-config/pull/5#discussion_r1775341759) so I can create an upstream PR.

Reply:
If the stubs are needed for pre-commit mypy tests to pass, then `test` seems appropriate to me.  I can see the case for `dev` as well, but I don't think it makes a huge difference one way or the other.

Reply:
Sorry I missed responding to that question.  I agree that the type requirements should be separated out.

See:
- 572b7f1ad9989f2553444885e7fa01ca4e8d5263

Reply:
I created cisagov/skeleton-python-library#141 to address this upstream, but the `setup-env` script still does not run successfully there even with the changes from cisagov/skeleton-python-library#138.  Any ideas what needs to be done there?